shared/components/charts/highcharts/column-chart/column-chart.hooks.tsx

```diff
@@ -59,6 +63,7 @@ export function useColumnChartOptions({
           style: yAxisPrimaryStyle,
         },
         gridLineColor: "#4C4C5C",
+        visible: isYAxisVisible,
```

Comment:
Maybe better to have a `yAxis` object prop to spread in case we want to add more properties later?

Reply:
Yes right! done